napari/layers/points/points.py

```diff
@@ -115,6 +98,19 @@ class Points(Layer):
         of the specified property that are mapped to 0 and 1, respectively.
         The default value is None. If set the none, the clims will be set to
         (property.min(), property.max())
+    border_width : float, array
+        Width of the symbol border in pixels.
+    border_width_is_relative : bool
+        If enabled, border_width is interpreted as a fraction of the point size.
+    cache : bool
+        Whether slices of out-of-core datasets should be cached upon retrieval.
+        Currently, this only applies to dask arrays.
+    canvas_size_limits : tuple of float
+        Lower and upper limits for the size of points in canvas pixels.
+        experimental_clipping_planes : list of dicts, list of ClippingPlane, or ClippingPlaneList
```

Comment:
```suggestion
    experimental_clipping_planes : list of dicts, list of ClippingPlane, or ClippingPlaneList
```
This one is a kwarg so it is too deeply intended